package.json

```diff
@@ -13,8 +13,8 @@
     "start": "run-p start:main start:worker",
     "start:main": "ts-node-dev --inspect=0.0.0.0:9229 --respawn --transpile-only --rs src/main",
     "start:worker": "ts-node-dev --inspect=0.0.0.0:9230 --respawn --transpile-only --rs src/worker",
-    "start:production": "run-p start:main:production start:worker:production",
-    "start:main:production": "node .",
+    "start:production": "run-p start:WebServer:production start:worker:production",
+    "start:WebServer:production": "node .",
```

Comment:
WebServer is the name of the default Micros group and there is no way to change it. So I had to rename the default command so the docker command would be able to pick it up.  I know it looks ugly now, so I am open for suggestions

Reply:
wait, so there's no way of changing the default in micros and it's using Pascal-case? :|

Reply:
I was thinking we could introduce some kind of "environment variable mapping" in the Dockerfile. That would map all Micros-specific env vars like this one to generic env vars that are used by the app. Maybe you can look into it if something like this is possible?

Reply:
I was initially thinking of something like that, but I afraid it can overcomplicate the situation.

However what can we do - map MICROS_GROUP to a nodejs command:

WebServer: "start:main:production"
Worker:    "start:worker:production"

Anything else - fail the start

Reply:
nah, I'd want to separate the app from the deployment as much as possible.  I agree that having it in the Dockerfile would make more sense.  Something like `CMD ["sh", "-c", "CMD=$(echo \"${MICROS_GROUP}\" | tr '[:upper:]' '[:lower:]' | sed 's/webserver/main/'); npm run start:$CMD:production"]` as the command for the dockerfile.  I just tested this in alpine docker and it seems to work.

Reply:
Let's keep the script, it is easier to read

Reply:
What script?

Reply:
`bin/start-server-micros.sh`